docs/README.md

```diff
@@ -415,9 +415,16 @@ An overview of the configurable parameters and their default value. These values
     Enables integration between Alfred Telemetry and Alfresco Micrometer metrics. If enabled, all Alfresco metrics 
     will automatically be written to the Alfred Telemetry registries.
 
-* `alfred.telemetry.alfresco-integration.include-default-alfresco-registry=true`  
-    In Alfresco 6.1, Alfresco itself has a Micrometer registry to write metrics to. This property configures if 
-    all metrics (Alfred Telemetry + Alfresco default) need to be written to Alfresco's global registry.
+* `alfred.telemetry.alfresco-integration.use-default-alfresco-registry=false`  
+    Since Alfresco 6.1, Alfresco itself provides a Micrometer meter registry. More specifically:
+    Alfresco provides a `PrometheusMeterRegistry`. With this property it is possible to indicate that Alfred Telemetry
+    should use the registry provided by Alfresco as it's Prometheus registry.  
+    The only advantage of using the default Alfresco registry is that metrics registered by Alfred Telemetry will
+    be available in the Alfresco scrape endpoint (`/alfresco/s/prometheus`). However this default 
+    registry is managed by Alfresco, hence that means no customizations like e.g. common tags are 
+    applied to the Prometheus metrics.  
+    If the default Alfresco registry is used, we should disable the prometheus registry initialized 
```

Comment:
Instead of "we should disable... " use more active language to make stronger impression to the reader that they have to take action, e. g. ", the prometheus registry must be disabled"